### ISSUE TITLE
First run

### ISSUE THREAD

**Issue by mzechmeister** [3fon3fonov/exostriker#3]
After uncommenting batman, i have it running!
![screenshot_20181119_001437](https://user-images.githubusercontent.com/32583239/48679495-4712cd80-eb91-11e8-9819-18baf92a1262.png)

The phase folding plot shows nothing.
How can I add a second planet with automatic start values?

**Comment by 3fon3fonov** [comment]
Something is wrong, post the Stdout/Stderr text (see top-right tab).

Why you had to comment batman? 

Also make sure the bash-script "check_your_system_linux_debian.sh" you got "Yes" everywhere.

**Comment by 3fon3fonov** [comment]
Mathias please with Python 3 pay attention to this warning in "README_for_installation":

 
# However, some modification of the RVmod object must be done when used with Py3
# because it cannot be "pickled" (i.e. copied via "copy" or "dill"). 
# This is peculiar because it works fine with Python2!
#
# On top of that there is some problem with piping the Python3 print() functions,
# which I haven't resolved yet. As a result the stdout is not printed in the
# "Stdout/Stderr" GUI tab, but directly in the terminal shell from which you 
# start the gui.


See my issue report for dill with Py3:

https://github.com/uqfoundation/dill/issues/293

I am sure thats your problem!  Fix it and the tool will work with Py3. This is to be resolved ASAP.

**Comment by mzechmeister** [comment]
> Something is wrong, post the Stdout/Stderr text (see top-right tab).

Empty!

> If you are running Python3 (which I think you do) the Stdout/Stderr
> is not active. Look for errors in the terminal from where you started the gui.

From terminal
```
===== 2018.11.19 23:11:29 =====
Traceback (most recent call last):
  File "gui.py", line 1126, in <lambda>
    self.button_init_fit.clicked.connect(lambda: self.optimize_fit(0))
  File "gui.py", line 881, in optimize_fit
    self.update_plots()                   
  File "gui.py", line 624, in update_plots
    self.update_extra_plots()
  File "gui.py", line 743, in update_extra_plots
    self.phase_plots(1)   
  File "gui.py", line 776, in phase_plots
    ph_data,ph_model = rv.phase_planet_signal(fit,ind)
  File "/home/zechmeister-local/python/trifon/RV_mod.py", line 524, in phase_planet_signal
    copied_obj = dill.copy(obj) 
  File "/usr/lib/python3.6/site-packages/dill/_dill.py", line 239, in copy
    return loads(dumps(obj, *args, **kwds))
  File "/usr/lib/python3.6/site-packages/dill/_dill.py", line 316, in loads
    return load(file, ignore)
  File "/usr/lib/python3.6/site-packages/dill/_dill.py", line 304, in load
    obj = pik.load()
  File "/usr/lib/python3.6/site-packages/dill/_dill.py", line 636, in _create_filehandle
    if position > current_size:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```
 
> I am sure thats your problem! Follow my fix in that link and the tool will work with Py3.

For me it was `/usr/lib/python3.6/site-packages/dill/_dill.py`. Error message is gone, but still empty stdout wigdet.

**Comment by 3fon3fonov** [comment]
> Error message is gone, but still empty stdout wigdet.

I need to fix that tab for Py3, I know. For now everything with Py3 is piped to the terminal. 
I will try to find a fix these days.

**Comment by 3fon3fonov** [comment]
Ok now I added OpenSuse installer which works perfectly fine!
Tested on a clean OpenSuse leap 15, installs everything one may need on Py2 and Py3.

For Py3, at the time this is written, the "dill" problem (see above) is still present. For Py3 one still needs to make the fix in the _dill.py 

I may add all the source codes (dill, batman, emcee, celerite, etc.) in the ./deps
directory and run a Local-install. This however is with low priority for now!
